src/styles/custom-components.less

```
@@ -212,6 +212,11 @@
     & [data-placeholder] {
         position: relative;
 
+        // ensures markdown placeholder wraps expectedly
+        &.markdown {
```

I'd scope this in a little further even

```suggestion
        &.s-code-block.markdown {
```